src/users/create_or_modify.rs

```diff
@@ -0,0 +1,108 @@
+//! Endpoint to create or modify a user account.
+
+pub use super::common::UserDetails;
+use ruma::{
+    api::{
+        client::{r0::contact::get_contacts::ThirdPartyIdentifier, Error},
+        ruma_api,
+    },
+    UserId,
+};
+
+pub mod v2 {
+    //! [GET /_synapse/admin/v2/users/:user_id](https://github.com/matrix-org/synapse/blob/master/docs/admin_api/user_admin_api.rst#create-or-modify-account)
+    use super::*;
```

Comment:
Please move `v2` into its own file and move the imports from above into it instead of using `super::*`.